modules/gatsby-theme-ocular/bin/ocular.js

```diff
@@ -136,7 +136,7 @@ const commands = {
           const file = readFileSync(`${TEMPLATE_DIR}/${filename}`);
```

Comment:
Can we delete the commented out code above?  Would be good to clean this file up to make it less confusing, unless these commented out snippets are critical for debugging.